src/MaterialDesignThemes.Wpf/Converters/FloatingHintTextBlockMarginConverter.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace MaterialDesignThemes.Wpf.Converters;
 
-internal class FloatingHintTextBlockMarginConverter : IMultiValueConverter
+public class FloatingHintTextBlockMarginConverter : IMultiValueConverter
```

Comment:
I propose moving this inside of a new `MaterialDesignThemes.Wpf.Converters.Internal` namespace.